### ISSUE TITLE
[Keyboard accessibilty] pressing tab tab... in fullscreen mode in paused state does not let focus go out

### ISSUE THREAD

**Issue by gupta816** [sampotts/plyr#2654]
### Expected behaviour
Keyboard accessibilty should work properly in fullscreen mode in paused state
### Actual behaviour
Keyboard accessibilty should does not work properly in fullscreen mode in paused state

### Steps to reproduce
1. Goto plyr.io
2. Enter fullscreen (player)
3. Open developers tool
4. Pause the video
5. Press tab tab tab..........
(Notice the tab focus does not goes to developers tool's items in paused state , but if you play the video it works fine.

### Environment

- Browser: Google Chrome
- Operating System: Windows 11
